src/main/java/org/opensearch/knn/plugin/script/KNNScoreScriptLeafFactory.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.plugin.script;
+
+import java.io.IOException;
+import java.util.Locale;
+import java.util.Map;
+
+import org.apache.lucene.index.LeafReaderContext;
+import org.apache.lucene.search.IndexSearcher;
+import org.opensearch.knn.plugin.stats.KNNCounter;
+import org.opensearch.script.ScoreScript;
+import org.opensearch.search.lookup.SearchLookup;
+
+/*
+    * A factory that creates KNNScoreScriptLeafFactory objects. The factory is responsible for parsing the parameters
```

Comment:
please fix formatting, there are some extra spaces at the begging of the line

Reply:
Thanks Martin, updated.